content/introduction/_index.md

```diff
@@ -5,30 +5,45 @@ draft: false
 weight: 1
 ---
 
-{{<title>}} is a vendor-agnostic single distribution of libraries to provide observability for your microservices and monoliths alike. OpenCensus is an open source community developed project. Its origins stem from the rewrite of what has been Google's observability systems for the past 10 years.
-
-OpenCensus is implemented in a plethora of languages and it allows you to singly collect metrics and traces
-and then export them to a variety of backends.
+{{<title>}} makes getting critical telemetry out of your services easy and automatic. Its origins stem from the rewrite of what has been Google's observability systems for the past 10 years. OpenCensus currently provides libraries for a number of languages that allow you to capture, manipulate, and export metrics and distributed traces to the backend(s) of your choice.
```

Comment:
Please keep the mention of "vendor-agnosticity", being "an open source community developed project" plus "for microservices and monoliths alike". I've directly interacted with many people who just think that this is an entirely Google project but also that they don't care about microservices.

Reply:
```suggestion
{{<title>}} makes getting critical telemetry out of your services easy and automatic. OpenCensus currently provides libraries for a number of languages that allow you to capture, manipulate, and export metrics and distributed traces to the backend(s) of your choice. OpenCensus works great with all software systems, from client applications, large monoliths, or highly-distributed microservices. OpenCensus started at Google but is now developed by a broad community of service developers, cloud vendors, and community contributors.
```

Reply:
Nice, SGTM!